src/components/main-dashboard/weather-details/Sun.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react';
+import { useWeather } from '../../WeatherProvider';
+import { getTimeNoSec } from '../../../utilities/helperFunctions';
+import sunriseImage from '../../../images/sunrise.svg';
+import sunsetImage from '../../../images/sunset.svg';
+
+export default function Sun() {
+  const { currentWeather } = useWeather();
+  const [weather, setWeather] = useState({});
+
+  useEffect(() => {
+    if (currentWeather) {
+      const { sys } = currentWeather;
+      const sunriseTime = getTimeNoSec(sys.sunrise);
+      const sunsetTime = getTimeNoSec(sys.sunset);
+      setWeather({
+        sunrise: sunriseTime,
+        sunset: sunsetTime,
+      });
+    }
+  }, [currentWeather]);
+
+  if (Object.keys(weather).length === 0) {
+    return null;
+  }
+  return (
+    <div className="sun col-4 border border-primary rounded pt-1">
```

Comment:
Yeah I see that too. I used a width of 90px on the sun styling and so that may be why. The units are not responsive. Let me see if using rem or some other units may help.

Reply:
When I use those same dimensions, there isn't any overlap regarding the sun component. Can you screenshot and post what it looks like on discord.